code/_helpers/logging/debug.dm

```diff
@@ -67,7 +67,12 @@
 
 /// Logging for world/Topic
 /proc/_log_topic(text)
-	WRITE_LOG(GLOB.config.logfiles["world_game_log"], "TOPIC: [text]")
+	WRITE_LOG(GLOB.config.logfiles["topic_log"], "TOPIC: [text]")
+
+// Logging for HTTP Requests
```

Comment:
```suggestion
/// Logging for HTTP Requests
```

Reply:
We dont have a style requirement there

Reply:
It's not a style requirement, it's how DMdoc works, with only 2 slashes it's not a DMdoc but a simple comment, which means the function will not give you a tooltip when you hover it in VSC